docs/VnVPlan/VnVPlan.tex

```diff
@@ -150,10 +150,57 @@ \section{Plan}
 
 \subsection{Verification and Validation Team}
 
-\wss{Your teammates.  Maybe your supervisor.
-  You should do more than list names.  You should say what each person's role is
-  for the project's verification.  A table is a good way to summarize
-this information.}
+The verification and validation team will consist of members from the
+development team as well as members from the project stakeholders. A summary of
+the members and their roles are given in the following table:
+\setlength{\arrayrulewidth}{0.5mm}
+\setlength{\tabcolsep}{18pt}
+\renewcommand{\arraystretch}{1.5}s
+\begin{tabular}{ | m{5cm} | m{9cm} | }
+  \hline
+  \textbf{Names and Role} & \textbf{Responsibilites} \\
+  \hline
+  \textbf{Mitchell Hynes, Kyle D'Souza, Richard Fan, Akshit Gulia, Rafeed
+  Iqbal} - Development Team &
+  \begin{itemize}
+    \item Responsible for verification of implementation details and
+      adherance of the system to the SRS. The development team will own the
+      most of the verification activities pertaining to the software. This
+      includes activities such as developing verification plans, code reviews,
+      unit testing, integration testing, and SRS verification.
+
+  \end{itemize}\\
+  \hline
+  \textbf{Matthew Yakymyshyn} - Stakeholder from the City of Hamilton &
+  \begin{itemize}
+    \item Responsible for the verification of the system. They will be
+      responsible for ensuring, and determining to what degree the
+      system fulfils its intended purpose.
+      Some examples of the activities that they would be responsible for
+      would be SRS validation and acceptance testing.
+
+  \end{itemize}\\
+  \hline
+  \textbf{Tarnveer Takhtar, Matthew Bradbury, Harman Bassi, Kyen So}
```

Comment:
I'm not 100% sure if our peer reviewers should be mentioned here, but approving regardless as it shouldn't be a big deal